examples/plot_anomaly_bench.py

```diff
@@ -1,19 +1,18 @@
 """
-====================================================================
-Comparison of LocalOutlierFactor and IsolationForest performances on 
-classical anomaly detection datasets using ROC curves
-====================================================================
-Benchmark plots of LocalOutlierFactor (LOF) and IsolationForest (IF) on 
-classical anomaly detection datasets. Note that LocalOutlierFactor is not 
-meant to predict on a test set by default and its performance is assessed 
-in an outlier detection context:
+===================================
+Anomaly detection's ROC benchmarks
+===================================
+Comparison of LocalOutlierFactor and IsolationForest performances on
+classical anomaly detection datasets using ROC curves. Note that
+LocalOutlierFactor is not meant to predict on a test set by default and its
```

Comment:
in fact I am just realizing now that we don't need the sentence about LOF here. However we should say that it is an outlier detection benchmark and keep the evaluation description.